CaptureSDK/GiniCaptureSDKPinning/Package-release.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "GiniCaptureSDKPinning",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v12), .macOS(.v10_13)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "GiniCaptureSDKPinning",
+            targets: ["GiniCaptureSDKPinning"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(name: "GiniBankAPILibraryPinning", url: "https://github.com/gini/bank-api-library-pinning-ios.git", exact.("0.0.1")),
+        .package(name: "GiniCaptureSDK", url: "https://github.com/gini/capture-sdk-ios.git", exact.("0.0.1")),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        
+        .target(
+            name: "GiniCaptureSDKPinning",
+            dependencies: ["GiniBankAPILibraryPinning"]),
```

Comment:
"GiniCaptureSDK" should be also in the dependencies. I added GiniCaptureSDKPinning to a test app and the GiniCaptureSDK was not added to the project. I had to add it separately.

But even then it didn't work because the pinning package couldn't find the Capture SDK:
<img width="1096" alt="Screenshot 2021-11-04 at 14 13 36" src="https://user-images.githubusercontent.com/13149498/140319665-9dfd2932-8c07-4764-972d-64216f3dd8b0.png">

Reply:
Thank you for testing it 🙈